Source/App/EbAppConfig.c

```diff
@@ -493,145 +460,131 @@ void EbConfigCtor(EbConfig_t *configPtr)
     configPtr->maxDisplayMasteringLuminance         = 0;
     configPtr->minDisplayMasteringLuminance         = 0;
 
-    configPtr->switchThreadsToRtPriority            = EB_TRUE;
-    configPtr->fpsInVps                             = EB_TRUE;
-    configPtr->unrestrictedMotionVector             = EB_TRUE;
-
     // Encoding Presets
-	configPtr->encMode                              = 9;
-    //configPtr->latencyMode                        = 0; // Deprecated
+    configPtr->encMode                              = 7;
+    //configPtr->latencyMode                          = 0; // Deprecated
+
     configPtr->speedControlFlag                     = 0;
 
     // Bit-depth
-    configPtr->encoderBitDepth                          = 8;
-    configPtr->compressedTenBitFormat			        = 0;
-    configPtr->encoderColorFormat                       = EB_YUV420;
+    configPtr->encoderBitDepth                      = 8;
+    configPtr->compressedTenBitFormat               = 0;
```

Comment:
Remove tab (line 471)

Reply:
It doesn't show as a tab.

Reply:
I'm seeing a tab after compressedTenBitFormat
(not sure how to paste what I'm seeing